bundles/Billing/Resources/config/services.yaml

```diff
@@ -2,6 +2,8 @@ services:
     _defaults:
         autowire: true      # Automatically injects dependencies in your services.
         autoconfigure: true # Automatically registers your services as commands, event subscribers, etc.
+        bind:
+            $boundToAccounts: '%env(json:BILLING_TO_ACCOUNTS)%'
```

Comment:
Depending on the complexity of the json expression in `BILLING_TO_ACCOUNTS`, it may be better to use something like `%env(json:file:resolve:BILLING_TO_ACCOUNTS_FILE)%` and put the data in a local file.